react/features/notifications/components/web/styled.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
```

Comment:
This is all good, but instead of styled components I would use JSS with our theme and tokens, maybe we need to use some of them.